CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -529,7 +529,7 @@ def appendFixColumns(ws, baseColumn, defaultColumnDict):
     return fixedColumnDict
 
 
-def validate_workbook(file_obj, messages, args=None):
+def validate_workbook(file_obj, messages=[], args=None):
```

Comment:
https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/